themes/custom/droopler_theme/scss/components/_d_title.scss

```diff
@@ -0,0 +1,26 @@
+// Big page header with h1 title.
+// Accepts header-name, background color, text color, padding top, padding bottom and header margin bottom.
+@mixin listing-main-header($header-name, $background-color, $text-color, $padding-top, $padding-bottom, $header-margin-bottom) {
+  .page-wrapper .main-header-wrapper
+  {
+
+    background-color: $background-color;
+
+    #{$header-name} {
+      color: $text-color;
+      padding-top: $padding-top;
+      padding-bottom: $padding-bottom;
+      text-transform: none;
+      text-align: center;
+
+      &:after {
+        @include decoration-header-line-main-center;
+      }
+
+      h1 {
+        color: $text-color;
+        margin-bottom: $header-margin-bottom;
+      }
+    }
+  }
+}
```

Comment:
i think that it would be better to move this file to some other directory than `components`,
for example `mixins`
then, when we attach this file to style.scss (the one to rule them all), you wouldn't have to import it every time you want to use it in some component..

take a look here: 
![image](https://user-images.githubusercontent.com/37533838/69437024-cb6ad400-0d42-11ea-86c4-4b6f4a39d30c.png)
i think it's quite good place to add there `@import mixins/title`

after that we would get rid of @import per component.

WDYT @grzegorz-pietrzak-droptica

Reply:
True, I'll fix it and merge.